index.rst

```diff
@@ -11,14 +11,78 @@ Bharat-Sim's documentation!
    :caption: Contents:
 
 
-
-Indices and tables
-==================
-
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
 
-How does this lool?
+
+
+
+SIR Model with Agents
+=====================
+
+The Bharat-sim framework allows you to build agent-based models. Agents, are essentially the main component of the simulation. They are entities which are defined with a bunch of parameters and certain rules which they have to obey as the simulation runs.
+
+In the purview of the SIR Model, the agents are people, which go about their daily lives and follow regular schedules. Here, the parameters defining an agent are a unique agent id, age, their infection status, the time for which they remain infected and so on. The simulation allows you to introduce new parameters as and when required.
+
+The rules governing the behaviour of all agents include the particular schedules which they follow and, the functions which allow the agents to transit from one compartment to the other.
+
+Schedules
+---------
+
+The current schedules of all the agents are stored in the **create12HourSchedules** function. The default function has three schedules:-
```

Comment:
You are writing this as if you are _describing_ the code that already exists. Don't do this, it's better to write it as if the reader is writing a new SIR model from scratch. Refer to [this link](https://gama-platform.github.io/wiki/PredatorPrey) for inspiration.

Reply:
I thought when people download Bharatsim for the first time, it will have some built-in examples like the SIR Model. So I thought it's better to explain what already exists and then describe how you can modify the code.

Reply:
This is true, however, I feel very strongly that simply reading what certain functions in a prepackaged code do doesn't really help one _use_ the code. The best thing to do would be to explain _what_ needs to be done, and _how_ we plan on going about doing it. _Then_ we explain the code.

The method you're suggesting was _sort of_ how I did it with you guys, but remember that you had access to me to answer all of your questions after the sessions: the people reading this documentation will not have any such access, so they need to be more comfortable with the material than you were, initially.

Reply:
Yeah that makes sense. We will start by explaining what needs to be done in the model.